packages/web/src/components/gcds-card/gcds-card.css

```diff
@@ -0,0 +1,128 @@
+@layer reset, defaults, slot, link, state.hover, state.focus;
+
+@layer reset {
+  :host * {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+}
+
+@layer defaults {
+  :host {
+    display: block;
+  
+    .gcds-card {
+      border: var(--gcds-card-border);
+      border-radius: var(--gcds-card-border-radius);
+      background-color: var(--gcds-card-background-color);
+      display: block;
+      padding: var(--gcds-card-padding);
+      height: 100%;
+      color: var(--gcds-card-color);
+      position: relative;
+
+      > *:not(slot):not(.gcds-card__spacer) {
+        display: block;
+        margin: var(--gcds-card-margin);
+      }
+
+      .gcds-card__image {
+        width: 100%;
+      }
+
+      .gcds-card__tag {
+        font: var(--gcds-card-tag-font);
+        color: var(--gcds-card-tag-color);
+      }
+  
+      .gcds-card__title,
+      .gcds-card__title > a {
+        color: var(--gcds-card-title-color);
+        font: var(--gcds-card-title-font);
+        text-underline-offset: var(--gcds-card-title-text-underline-offset);
+        text-decoration-color: currentColor;
+        text-decoration-thickness: var(--gcds-card-title-text-decoration-thickness);
+        width: fit-content;
+      }
+  
+      .gcds-card__description {
+        max-width: 65ch;
```

Comment:
Can we use a container token here? The new Lg container token will be `62rem`, you can see a list of the changed tokens on this branch:

https://github.com/cds-snc/gcds-tokens/blob/add-nav-tokens/tokens/global/container/base.json

Reply:
I set the description width to `65ch` to match how we limit the line length for accessibility since the `description` could be a couple lines long.

Do you think it is still worth it to switch to a container token?

Reply:
Oh sorry, my bad! I read `65rem` instead of `65ch` - definitely keep it as `65ch` pls, but can we still put it in a token?

Reply:
Will do!